hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/upgrade/AbstractLayoutVersionManager.java

```diff
@@ -107,6 +121,10 @@ public void doFinalize(Object param) {
     }
   }
 
+  public void completeFinalize(Object param) {
+    currentUpgradeState = Finalized;
```

Comment:
param not used here.

Reply:
yup. eventually I would remove it if not used.